modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -610,9 +610,9 @@ spec:
  hugepages:
   defaultHugepagesSize: "1G"
   pages:
-  -size: "1G"
-   count: 16
-   node: 0
+  - size: "1G"
```

Comment:
Hi @johnwilkins @StephenJamesSmith - Do you know who in QE can verify this change? Thanks!